registration/registration-client/src/main/java/io/mosip/registration/constants/RegistrationUIConstants.java

```diff
@@ -37,9 +33,8 @@ public static void setBundle() {
 	public static String getMessageLanguageSpecific(String key) {
 		try {
 			return bundle.getString(key);
-		} catch (Exception runtimeException) {
-			LOGGER.error("REGISTRATION_UI_CONSTANTS", APPLICATION_NAME, APPLICATION_ID,
-					ExceptionUtils.getStackTrace(runtimeException));
+		} catch (Exception exception) {
+			LOGGER.error("Exception in finding property in ResourceBundle " + exception);
```

Comment:
can you replace + with , like below
LOGGER.error("Exception in finding property in ResourceBundle ", exception);

Reply:
Modified as suggested